CHANGELOG.md

```diff
@@ -38,6 +38,15 @@
 
 [Complete Changeset](https://github.com/jacobwilliams/json-fortran/compare/7.1.0...HEAD)
 
+**Enhancements:**
+
+- Allow linking to JSON-Fortran from GFortran programs using
+  [OpenCoarrays](https://github.com/sourceryinstitute/OpenCoarrays#readme)
+  as the coarray runtime implementation. Use the
+  `-DJSON_FORTRAN_USE_OpenCoarrays:BOOL=ON` option to cmake to enable
+  this. (NOTE: The fact that this is required may be a bug in
+  GFortran.)
```

Comment:
@jacobwilliams how's this for the explanation? If you prefer, I can remove this option and users can manually tweak CMake flags if they need to.

Reply:
That sounds fine. I don't mind keeping it if it's useful.